scripts/update-version.js

```diff
@@ -0,0 +1,105 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+const RELAY_NPM_PACKAGES = [
+  'babel-plugin-relay',
+  'react-relay',
+  'relay-compiler',
+  'relay-runtime',
+  'relay-test-utils',
+  'relay-test-utils-internal',
+];
+
+const CARGO_FILES = [
+  `compiler/crates/relay-bin/Cargo.toml`,
+  `compiler/Cargo.lock`,
+];
+
+/**
+ * Updates the version of relay across all package.json files
+ * Based on previous internal version @see https://gist.github.com/captbaritone/3351af94fb0821b93e26e23fbbc0c4c5
+ * @param {string} version The version to update to
+ * @example
+ * node scripts/update-version.js 1.0.0
+ */
+(function main() {
```

Comment:
Is this OK? I'm not a fan of this IIFE, any recommendations? Should I call `main()` at the end I'm not sure if I like either 😀 ? Or the goal was to keep it "compatible" with "js1" (no idea about it :D ).

```js
exports.command = 'update-version VERSION';
exports.describe = 'Updates the version of relay across all package.json files';
exports.handler = argv => {
   //...
}
```